packages/lodestar/src/chain/blocks/processor.ts

```diff
@@ -58,6 +58,8 @@ export class BlockProcessor implements IService {
 
   public async start(): Promise<void> {
     const abortSignal = this.controller.signal;
+    // An error in this pipe should panic the process, so it's not handled
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
     pipe(
```

Comment:
```suggestion
   void pipe(
```

Reply:
That's some crazy (and nice) javascript voodoo magic. For a new-comer I'm not sure what's more confusing `void` or `eslint-disable-next-line @typescript-eslint/no-floating-promises`